Babelify requires Babel 7.x (@babel/core) in Tutorial > Gulp page

How to include Babel into Gulp on the  [_Tutorial_ > _gulp_ page](https://www.typescriptlang.org/docs/handbook/gulp.html) is old. An error occurred in the instruction. 

```
proj/node_modules/babelify/index.js:16
  throw err;
  ^

Error: Cannot find module '@babel/core'
 babelify@10 requires Babel 7.x (the package '@babel/core'). If you'd like to use Babel 6.x ('babel-core'), you should install 'babelify@8'.
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (proj/node_modules/babelify/index.js:9:11)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

So I suggest updating the package installation to using `@babel/core` instead of `babel-core`. Or downgrade Babelify to 8.x.

--- a/pages/tutorials/Gulp.md
+++ b/pages/tutorials/Gulp.md
@@ -398,7 +398,7 @@ Like Uglify, Babelify mangles code, so we'll need vinyl-buffer and gulp-sourcema
 By default Babelify will only process files with extensions of `.js`, `.es`, `.es6` and `.jsx` so we need to add the `.ts` extension as an option to Babelify.
 
 ```shell
-npm install --save-dev babelify babel-core babel-preset-es2015 vinyl-buffer gulp-sourcemaps
+npm install --save-dev babelify@8 babel-core babel-preset-es2015 vinyl-buffer gulp-sourcemaps
 ```
 
 Now change your gulpfile to the following:

